db/migrate/20230124144929_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user
```

Comment:
- [ ] As per this [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_1.md#:~:text=Table%20and%20column%20names%20should%20match%20the%20ERD%20diagram), Kindly create the foreign key between users table and comments table as `author_id` not `user_id` like you did here.
[Hint]: you can do it like that:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
I just updated that `t.references` line and I used your suggestion in my file, thank you.